data/sidebar_docs.yaml

```diff
@@ -1,78 +1,82 @@
-- title_en: Robonomics Home Assistant Integration
+- title_en: Smart Home
   items:
     - title_en: Overview
       link: /docs/robonomics-smart-home-overview
       withoutNav: true
-    - title_en: Pre-installed Image For Raspberry Pi
+    - title_en: Install Smart Home
+      link: /docs/install-smart-home
+    - title_en: Home Assistant Init
+      link: /docs/hass-init
+    - title_en: Add devices
       items:
-        - title_en: Installation
-          link: /docs/hass-image-install
-        - title_en: Home Assistant Init
-          link: /docs/hass-init
-        - title_en: Zigbee2MQTT Adapter
-          link: /docs/zigbee-to-mqtt-image
-        - title_en: Subscription Activate
-          link: /docs/sub-activate
-          topic: Pre-installed Image For Raspberry Pi
-        - title_en: Robonomics Integration Setup
-          link: /docs/robonomics-hass-integration
-          topic: Pre-installed Image For Raspberry Pi
-          next:
-            - title: Global Administration
-              link: /docs/global-administration
-    - title_en: Upgrade Home Assistant OS
+        - title_en: Zigbee Devices
+          link: /docs/zigbee-devices
+        - title_en: Virtual Devices
+          link: /docs/virtual-devices
+    - title_en: Subscription Activate
+      link: /docs/sub-activate
+      topic: Smart Home
+    - title_en: Robonomics Integration Setup
+      link: /docs/robonomics-hass-integration
+      topic: Smart Home
+      next:
+        - title: Global Administration
+          link: /docs/global-administration
```

Comment:
link from the `robonomics-hass-integration` article to Global Administration is broken